components/classes/branch.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/classes/branch",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Branch",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
+  "description": "For branch information such as ID, name, and open and close times.",
+  "definitions": {
+    "branch": {
+      "properties": {
+        "branch": {
```

Comment:
No need for this parent "branch" field here. This json-schema here is defining a whole new business entity (aka class) so the fields in the class should be root level fields that apply to all schemas based on this class. 

You often only create nested structures inside a mixin where the fields being added to the business entity have some top level context (parent field) they are associated with.

Reply:
Thanks Craig - that makes sense.  Can you take a look at these updates and confirm they are correct?